tests/tests_integration/test_api/test_simulators/test_simulators.py

```diff
@@ -0,0 +1,221 @@
+
+        assert len(integrations) > 0
+
+    def test_filter_integrations(self, cognite_client: CogniteClient, seed_resource_names) -> None:
+        all_integrations = cognite_client.simulators.integrations.list()
+        active_integrations = cognite_client.simulators.integrations.list(
+            filter=SimulatorIntegrationFilter(active=True)
+        )
+
+        filtered_integrations = cognite_client.simulators.integrations.list(
+            filter=SimulatorIntegrationFilter(simulator_external_ids=[seed_resource_names["simulator_external_id"]])
+        )
+
+        assert len(all_integrations) > 0
+        assert filtered_integrations[0].external_id == seed_resource_names["simulator_integration_external_id"]
+        # check time difference
+        assert filtered_integrations[0].active is True
+
+        assert len(active_integrations) > 0
+        assert len(filtered_integrations) > 0
+
+
+@pytest.mark.usefixtures(
+    "seed_resource_names",
+    "seed_simulator_model_revisions",
+)
+class TestSimulatorModels:
+    def test_list_models(self, cognite_client: CogniteClient, seed_resource_names) -> None:
+        models = cognite_client.simulators.models.list(
+            limit=5, filter=SimulatorModelsFilter(simulator_external_ids=[seed_resource_names["simulator_external_id"]])
+        )
+        assert len(models) > 0
+
+    def test_retrieve_model(self, cognite_client: CogniteClient, seed_resource_names) -> None:
+        model = cognite_client.simulators.models.retrieve(external_id=model_unique_external_id)
+        assert model is not None
+        assert model.external_id == model_unique_external_id
+
+    def test_list_model_revisions(self, cognite_client: CogniteClient, seed_resource_names) -> None:
+        revisions = cognite_client.simulators.models.list(
+            limit=5,
+            filter=SimulatorModelRevisionsFilter(model_external_ids=[model_unique_external_id]),
+        )
+        assert len(revisions) > 0
+
+    def test_retrieve_model_revision(self, cognite_client: CogniteClient, seed_resource_names) -> None:
+        model_revision = cognite_client.simulators.models.revisions.retrieve(
+            external_id=seed_resource_names["simulator_model_revision_external_id"]
+        )
+        assert model_revision is not None
+        assert model_revision.model_external_id == model_unique_external_id
+
+
+@pytest.mark.usefixtures("seed_resource_names", "seed_simulator_routine_revisions")
+class TestSimulatorRoutines:
+    def test_list_routines(self, cognite_client: CogniteClient) -> None:
+        routines = cognite_client.simulators.routines.list(limit=5)
+        assert len(routines) > 0
+
+    def test_list_routine_revisions(self, cognite_client: CogniteClient) -> None:
+        revisions = cognite_client.simulators.routines.revisions.list(limit=5)
+        assert revisions[0].configuration is not None
+        assert revisions[0].script is not None
+        assert len(revisions) > 0
```

Comment:
you are setting limit here, because the default one breaks the test, right? come on 😅

`limit: int = DEFAULT_LIMIT_READ,` this won't work for routine revisions since we allow max=20.